O/OpenCV/build_tarballs.jl

```diff
@@ -103,4 +116,4 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"7")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.8", preferred_gcc_version = v"9")
```

Comment:
Assumed that this line https://github.com/JuliaPackaging/Yggdrasil/blob/5d8815ce438387434c7a867f668cd6d4a0bfb01f/O/OpenCV/build_tarballs.jl#L114 needed to be aligned with the build_tarballs constraint, is this not the case?

Reply:
Then it should do what https://github.com/JuliaPackaging/Yggdrasil/blob/11ca09f0670f25fa03c340bda390ecb028d54633/L/libcxxwrap_julia/build_tarballs.jl does

Reply:
Thanks for the pointer! Think this is done. (The Julia versions used by libcxxwrap_julia are not all the latest point release, e.g. 1.6.3 instead of 1.6.7. Is this relevant? Should they be bumped everywhere?)